app/views/search/_form.html.erb

```diff
@@ -63,18 +63,16 @@ end
           <input type="text" class="field field-text wide" id="advanced-title" name="title" placeholder="Title" value=<%= params[:title] %>>
         </div>
 
-        <div class="field-wrap">
-          <label for="advanced-source" class="field-label">Source</label>
-          <select id="advanced-source" class="field field-select" name="source">
-            <option></option>
-            <option <%= "selected" if params[:source] == "Abdul Latif Jameel Poverty Action Lab Dataverse"%>>Abdul Latif Jameel Poverty Action Lab Dataverse</option>
-            <option <%= "selected" if params[:source] == "DSpace@MIT"%>>DSpace@MIT</option>
-            <option <%= "selected" if params[:source] == "Woods Hole Open Access Server"%>>Woods Hole Open Access Server</option>
-            <option <%= "selected" if params[:source] == "Zenodo"%>>Zenodo</option>
-            <option>ArchivesSpace@MIT (no data yet)</option>
-          </select>
-          NOTE: to allow selecting multiple sources we need to change the way we handle this in TIMDEX to treat source as an
-          "OR" search not "AND"
+        <div class="field-wrap list-checkboxes">
+          <%# https://www.w3.org/WAI/tutorials/forms/grouping/ %>
+          <fieldset>
+            <legend>Limit search to checked sources.</legend>
+
+            <%= source_checkbox('Abdul Latif Jameel Poverty Action Lab Dataverse', params) %>
+            <%= source_checkbox('DSpace@MIT', params) %>
+            <%= source_checkbox('Woods Hole Open Access Server', params) %>
+            <%= source_checkbox('Zenodo', params) %>
```

Comment:
Yup. I think it's a mess in ENV though as we'll need to keep it in sync. I think source control is appropriate, but not inline like this. I was just talking with Helen and Eric about non-secret config that should be the same in all envs (dev/test/stage/prod) and how ENV is messy for that.

In this case, we may end up wanting (for a period of time at least) an RDI deploy of this app and a non-RDI deploy which would lead to ENV being a good place for this list to live. Hmmm. I'm going to give it some thought. Thanks.